### DISCUSSION TITLE
freetz-ng as a repeater of another different (unkown not fritz!box) router

### DISCUSSION THREAD

**Discussion by Crazy4electrons** [Freetz-NG/freetz-ng#612] [Allgemein]
Hi every one, I'm new to freetz .
I want to ask help to extend my neighbours wifi. 
The current router being used is hauwei and i have a Fritz!box 7430 to use as the repeater.
on the default web api, I can set the router  as an IP repeater but only if the other router is a Fritz!box i assume, 
because the SSID is being shown in the list but is grey out.
please help.?
i already have a freetz installed on the Fritz!box 7430,so reinstall wont be a problem.
i just need to know which modules i should install and how to configure them...
thanks in aadvance

**Comment by fda77** [comment]
You could try openwrt: https://openwrt.org/toh/hwdata/avm/avm_fritz_box_7430
`+ no fritzos`
`- no dect, fon, isdn etc`
